lib/EntryHelpers.php

```diff
@@ -240,6 +240,7 @@ public static function extractCaseStudySummary(Entry $entry)
             'trailText' => $entry->caseStudyTrailText,
             'trailTextMore' => $entry->caseStudyTrailTextMore,
             'grantAmount' => $entry->caseStudyGrantAmount,
+            'grantId' => $entry->caseStudyGrantId ? $entry->caseStudyGrantId : null,
```

Comment:
:neckbeard: You can use the null coalesce thing here btw. `$entry->caseStudyGrantId ?? null`. Don't worry about changing it here though.